test/e2e/pods.go

```diff
@@ -533,32 +533,34 @@ var _ = Describe("Pods", func() {
 		}, 1)
 	})
 
-	It("should have monotonically increasing restart count", func() {
-		runLivenessTest(framework.Client, framework.Namespace.Name, &api.Pod{
-			ObjectMeta: api.ObjectMeta{
-				Name:   "liveness-http",
-				Labels: map[string]string{"test": "liveness"},
-			},
-			Spec: api.PodSpec{
-				Containers: []api.Container{
-					{
-						Name:    "liveness",
-						Image:   "gcr.io/google_containers/liveness",
-						Command: []string{"/server"},
-						LivenessProbe: &api.Probe{
-							Handler: api.Handler{
-								HTTPGet: &api.HTTPGetAction{
-									Path: "/healthz",
-									Port: util.NewIntOrStringFromInt(8080),
+	/*
+		It("should have monotonically increasing restart count", func() {
```

Comment:
better to use `PIt(` rather than commenting out. That'll let us see it's skipped.